day_two_guide/topics/increasing_docker_storage.adoc

```diff
@@ -103,12 +103,57 @@ volumes associated with container storage:
 +
 ----
 # docker-storage-setup
+----
++
+.. For *thin pool* setups, you should see the following output and can proceed to the next step:
++
+----
 INFO: Volume group backing root filesystem could not be determined
 INFO: Device /dev/xvdb is already partitioned and is part of volume group docker_vol
 INFO: Device node /dev/xvdd1 exists.
   Physical volume "/dev/xvdd1" successfully created.
   Volume group "docker_vol" successfully extended
 ----
++
+.. For *XFS* setups that use the Overlay2 file system, the above increase will not be visible.
```

Comment:
Should this say something like "will not be visible, so you must <do whatever you're doing in the following 3 steps?>?

Reply:
Yes, that would be helpful!